index.bs

```diff
@@ -0,0 +1,225 @@
+    1. Let <var>impactedFraction</var> be the area of the <a>initial containing block</a> of the <var>impactedRegion</var> divided by the area of the current <a>viewport</a>.
+    1. Set <var>layoutShift</var> to the <var>impactedFraction</var> multiplied by the <var>maxShiftFraction</var>.  
+    1. If <var>layoutShift</var> is greater than 0, create a new {{LayoutShift}} object |newEntry| and set its attributes as follows:
+        1. Set |newEntry|'s {{PerformanceEntry/name}} attribute to <code>"layout-shift"</code>.
+        1. Set |newEntry|'s {{PerformanceEntry/entryType}} attribute to <code>"layoutShift"</code>.
+        1. Set |newEntry|'s {{PerformanceEntry/startTime}} attribute to <a>current high resolution time</a>.
+        1. Set |newEntry|'s {{PerformanceEntry/duration}} attribute to 0.
+        1. Set |newEntry|'s {{LayoutShift/value}} attribute to <var>layoutShift</var>.
+    1. <a href="https://w3c.github.io/performance-timeline/#dfn-queue-a-performanceentry">Queue the PerformanceEntry</a> |newEntry| object.
+</div>
+
+NOTE: This computation ensures that the layout shift takes into account both the fraction of the viewport that has been impacted by layout stability as well as the greatest impact to any given element in the viewport.  This is to recognize that a large element that moves only a small distance may have a small impact on the perceived stability of the page.
+
+Identify an unstable element {#sec-identify-unstable-element}
+--------------------------------------------------------------
+
+<div algorithm="identify an unstable element">
+    When asked to <dfn export>identify an unstable element</dfn>, given <var>element</var> as input, run the following steps:
+
+    1. If <var>element</var>’s <a>previous frame starting point</a> is null, return false.
+    1. If <var>element</var>’s <a>previous frame starting point</a> does not equal <var>element</var>’s <a>current frame starting point</a> and they differ by 3 or more pixel units in the horizontal or vertical direction, return true.
+    1. Return false.
+
+NOTE: Updating an element's CSS transform doesn't cause it to be an unstable element.
+
+</div>
+
+Security & privacy considerations {#priv-sec}
+===============================================
+
+Layout instability bears an indirect relationship to <a href="https://w3c.github.io/resource-timing/">resource timing</a>, as slow resources could cause intermediate layouts that would not otherwise be performed. Resource timing information can be used by malicious websites for <a>statistical fingerprinting</a>.
+The layout instability API only reports instability in the current browsing context. It does not directly provide any aggregate instability scores. Developers can implement such aggregation manually, but browsing contexts with different <a href="https://html.spec.whatwg.org/multipage/origin.html#concept-origin">origins</a> would need to cooperate to share instability scores.
```

Comment:
"any aggregate instability scores" -> "any aggregation of instability scores across multiple browsing contexts"

Reply:
done.